src/config-import.js

```diff
@@ -108,14 +108,14 @@ export async function readConfig(configPath) {
     *fields() {
       const { fields } = presetsFile
       for (const [name, field] of Object.entries(fields)) {
-        if (!isRecord(field) || !hasOwn(field, 'key')) {
+        // is the hasOwn 'tagKey' check necessary?
+        if (!isRecord(field) || !hasOwn(field, 'tagKey')) {
```

Comment:
I don't think so. It's handled by `validate` below.

```suggestion
        if (!isRecord(field)) {
```